.github/workflows/realease-on-pr-merge.yaml

```
@@ -0,0 +1,47 @@
+        uses: actions/setup-node@v4
+        with:
+          node-version: 'lts/*'
+      - name: Get PR Labels
+        id: get_labels
+        run: |
+          LABELS=$(gh pr view ${{ github.event.pull_request.number }} --json labels --jq '.lables[].name')
+          echo "Labels: $LABELS"
+          echo "::set-output name=labels::$LABELS"
+      - name: Determine Version Bump
+        id: version_type
+        run: |
```

actionlint

shellcheck reported issue in this script: SC1046:error:2:1: Couldn't find 'fi' for this 'if'

(shellcheck)